Source/BoundaryConditions/PEC_Insulator.H

```diff
@@ -0,0 +1,174 @@
+     */
+    void ApplyPEC_InsulatortoBfield (std::array<amrex::MultiFab*, 3> Bfield,
+                                     amrex::Array<FieldBoundaryType,AMREX_SPACEDIM> const & field_boundary_lo,
+                                     amrex::Array<FieldBoundaryType,AMREX_SPACEDIM> const & field_boundary_hi,
+                                     amrex::IntVect const & ng_fieldgather, amrex::Geometry const & geom,
+                                     int lev, PatchType patch_type, amrex::Vector<amrex::IntVect> const & ref_ratios,
+                                     amrex::Real time);
+
+    /**
+     * \brief The work routine applying the boundary condition
+     *
+     * \param[in,out] field
+     * \param[in]     ng_fieldgather      number of guard cells used by field gather
+     * \param[in]     geom                geometry object of level "lev"
+     * \param[in]     lev                 level of the Multifab
+     * \param[in]     patch_type          coarse or fine
+     * \param[in]     ref_ratios          vector containing the refinement ratios of the refinement levels
+     * \param[in]     time                current time of the simulation
+     * \param[in]     split_pml_field     whether pml the multifab is the regular Efield or
+     *                                    split pml field
+     * \param[in]     E_like whether the field is E like or B like
+     * \param[in]     set_F_x_lo whether the tangential field at the boundary was specified
+     * \param[in]     set_F_x_hi whether the tangential field at the boundary was specified
+     * \param[in]     a_Fy_x_lo the parser for the tangential field at the boundary
+     * \param[in]     a_Fz_x_lo the parser for the tangential field at the boundary
+     * \param[in]     a_Fy_x_hi the parser for the tangential field at the boundary
+     * \param[in]     a_Fz_x_hi the parser for the tangential field at the boundary
+     * \param[in]     set_F_y_lo whether the tangential field at the boundary was specified
+     * \param[in]     set_F_y_hi whether the tangential field at the boundary was specified
+     * \param[in]     a_Fx_y_lo the parser for the tangential field at the boundary
+     * \param[in]     a_Fz_y_lo the parser for the tangential field at the boundary
+     * \param[in]     a_Fx_y_hi the parser for the tangential field at the boundary
+     * \param[in]     a_Fz_y_hi the parser for the tangential field at the boundary
+     * \param[in]     set_F_z_lo whether the tangential field at the boundary was specified
+     * \param[in]     set_F_z_hi whether the tangential field at the boundary was specified
+     * \param[in]     a_Fx_z_lo the parser for the tangential field at the boundary
+     * \param[in]     a_Fy_z_lo the parser for the tangential field at the boundary
+     * \param[in]     a_Fx_z_hi the parser for the tangential field at the boundary
+     * \param[in]     a_Fy_z_hi the parser for the tangential field at the boundary
```

Comment:
Could you add the docstrings for `field_boundary_lo` and `field_boundary_hi` as well?